test/scripts/e2e-cp.sh

```diff
@@ -184,13 +184,18 @@ export NUM_NODES=2
 
 FOCUS=$(echo ${@:1} | sed 's/ /\\s/g')
 
+# Ginkgo test timeout needs to be lower than both github's timeout and go test
+# timeout to be able to get proper Ginkgo output when it happens.
+TEST_TIMEOUT=${TEST_TIMEOUT:-180}
```

Comment:
so this is the timeout for a single test suite? so individual test suite max will take 180?